doc/tools/docgram/doc_grammar.ml

```diff
@@ -554,6 +560,10 @@ let level_regex = Str.regexp "[a-zA-Z0-9_]*$"
 let get_plugin_name file =
   if file = "user-contrib/Ltac2/g_ltac2.mlg" then
     "Ltac2"
+  (* todo: would be better if g_search.mlg has an "ssr" prefix *)
```

Comment:
All these paths start with `plugins/ssr` though.

Reply:
In my dev tools, I am almost always working with the simple filenames rather than a path.  Very very frequently I'm trying to pick from a list of mlg file names.  The other 3 ssr mlgs include "ssr" in the filename.

Reply:
OK, I thought this todo was for this particular code. Anyway, this code is probably going away soon, so not worth renaming IMHO.